### DISCUSSION TITLE
Sidebar with notebook files

### DISCUSSION THREAD

**Discussion by Elphikaus** [fonsp/Pluto.jl#1172] [Pluto development]
So I am working on Julia code in pluto and loving it. I open a notebook in a tab and then when I need a different project/script, I open a new tab. This worked fine.

I was wanting to see a top view of related files in one tab. I can not see an option for this.  It is great to use PlutoUI and be able to add a table of contents. But when I say working on a file of functions/struct/data/etc I get confused switching from tab to tab. 

A solution to this problem is of course the MIT  webpage: https://computationalthinking.mit.edu/Spring21/

This may be a mixture of server-side stuff, but on a local machine, how hard would this be to look at?

**Comment by fonsp** [comment]
I am not sure what you mean with a "top view of related files in one tab", can you edit your issue to clarify? Maybe add a screenshot of another program that provides what you mean

**Reply by Elphikaus** [reply]
![Screenshot 2021-05-18 at 15 13 44](https://user-images.githubusercontent.com/19248072/118667248-beba2580-b7eb-11eb-9e5d-605fa2158e20.jpg)

I know this is a very organised setting, but what if you had a side view of pluto notebooks/script/data/ in a given folder or environment.  

Does this make sense?

**Reply by fonsp** [reply]
Ah now I see what you mean. Good idea! Can you post this to our [suggestion box](https://github.com/fonsp/Pluto.jl/issues/182)?

**Reply by Elphikaus** [reply]
Yep, no problem

**Reply by BeastyBlacksmith** [reply]
I second this. This would also making switching between notebooks more convenient

**Reply by Elphikaus** [reply]
I know it has been ages since I requested this, but I just added it to the suggestion box.
